netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -198,6 +199,14 @@ private NettyTransportFactory(SocketAddress serverAddress,
       this.flowControlWindow = flowControlWindow;
       this.negotiator = negotiator;
       this.maxMessageSize = maxMessageSize;
+      if (serverAddress instanceof InetSocketAddress) {
```

Comment:
Should we make this a utility method in `GrpcUtil`?  Seems pretty generic, even if it's only used here.